src/rqt_launch/launch_widget.py

```diff
@@ -269,10 +269,10 @@ def save_settings(self, plugin_settings, instance_settings):
         pass
 
     def restore_settings(self, plugin_settings, instance_settings):
-#        if instance_settings.contains('splitter'):
-#            self._splitter.restoreState(instance_settings.value('splitter'))
-#        else:
-#            self._splitter.setSizes([100, 100, 200])
+        #        if instance_settings.contains('splitter'):
```

Comment:
Is this commented code needed?

Reply:
Is this method needed?

Reply:
Maybe it should be deleted in a separate PR. I didn't write this so I'm not sure but it would be worth looking through the history before removing commented out code. Hopefully, the author put a note about why it was commented out in the commit message.